src/genie/libs/parser/nxos/tests/ShowTrack/cli/equal/golden_output1_output.txt

```diff
@@ -0,0 +1,4 @@
+Track 1
```

Comment:
do you have any other output which has multiple tracks? and if yes, please add as `golden_output2_output.txt`

Reply:
@tahigash yes, i have added the 2nd golden output for the multiple track output example

Reply:
added the 2nd golden output for the multiple track